src/QMCWaveFunctions/SPOSet.h

```diff
@@ -281,7 +284,7 @@ class SPOSet : public QMCTraits
                                       const RefVector<ValueVector>& psi_v_list,
                                       const RefVector<GradVector>& dpsi_v_list,
                                       const RefVector<ValueVector>& d2psi_v_list,
-                                      const RefVector<ValueVector>& dspin_v_list) const;
+                                      DualMatrix<ComplexType>& mw_dspin) const;
```

Comment:
Update the documentation and mention that the device side is up-to-date.

Reply:
added documentation